rxbinding-support-v4/src/androidTest/java/com/jakewharton/rxbinding2/support/v4/widget/RxSwipeRefreshLayoutTest.java

```diff
@@ -36,28 +36,28 @@
   }
 
   @Test public void refreshes() throws InterruptedException {
-    RecordingObserver<Void> o = new RecordingObserver<>();
-    Subscription subscription = RxSwipeRefreshLayout.refreshes(view)
+    RecordingObserver<Notification> o = new RecordingObserver<>();
```

Comment:
I've converted `Void` to `Object` in 9f7f0a309f6934cf9e14a8a3044b072e0b7062e5 and in many others.
Do you think this matters? Should I change it?